benefits/core/migrations/0002_data.py

```diff
@@ -28,10 +28,22 @@ def load_data(app, *args, **kwargs):
     sbmtd_senior_type = EligibilityType.objects.create(
         name="senior", label="Senior Discount (SBMTD)", group_id=os.environ.get("SBMTD_SENIOR_GROUP_ID", "group4")
     )
+    sbmtd_mobility_pass_type = EligibilityType.objects.create(
+        name="mobility_pass",
+        label="Mobility Pass Discount (SBMTD)",
+        group_id=os.environ.get("SBMTD_MOBILITY_PASS_GROUP_ID", "group5"),
+    )
 
     PemData = app.get_model("core", "PemData")
 
-    server_public_key = PemData.objects.create(
+    mst_server_public_key = PemData.objects.create(
+        label="Eligibility server public key",
+        remote_url=os.environ.get(
+            "SERVER_PUBLIC_KEY_URL", "https://raw.githubusercontent.com/cal-itp/eligibility-server/dev/keys/server.pub"
+        ),
+    )
+
+    sbmtd_server_public_key = PemData.objects.create(
         label="Eligibility server public key",
         remote_url=os.environ.get(
             "SERVER_PUBLIC_KEY_URL", "https://raw.githubusercontent.com/cal-itp/eligibility-server/dev/keys/server.pub"
```

Comment:
Should these `SERVER_PUBLIC_KEY_URL` point somewhere else now? Create a separate `server.pub` files for SBMTD?

Reply:
Yeah I think that is what this task is trying to say:

> Need a per-verifier `SERVER_PUBLIC_KEY_URL` variable

In other words, we can't just have a single variable. We'd need `MST_SERVER_PUBLIC_KEY`, `SBMTD_SERVER_PUBLIC_KEY` etc.